infrastructure/local/docker-compose.hlb.yml

```diff
@@ -40,41 +43,34 @@ services:
       TIGERBEETLE_CLUSTER_ID: ${TIGERBEETLE_CLUSTER_ID-0}
```

Comment:
Maybe the `DATABASE_URL` could include username / password with something related to `happy-life-bank`?